docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,273 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+## Using NGINX Ingress Controller with Istio service mesh    
+
+NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds.
+This is accomplished using the special setting [use-cluster-ip](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream) for the backend.
+
+
+Here is a standard deployment of NGINX Ingress controller without a sidecar proxy injected into the pod.    
+
+{{< img src="./img/nginx_plain.png" alt="NGINX stand alone." >}}
+

```

Comment:
Notice that NGINX Ingress Controller is enumerating the pods of the backend service and balancing traffic directly to them, granting you full control of backend loadbalancer and stickyness behavior.
If your service pods supported TLS, then NIC could re-encrypt the traffic to them and provide end-to-end encryption.

But then if your services could do that, you might no be implementing a Service Mesh.

To begin, Istio needs to be installed into your cluster.